examples/cloud/src/App.tsx

```diff
@@ -0,0 +1,49 @@
+import { Refine } from "@pankod/refine-core";
+import {
+    notificationProvider,
+    Layout,
+    ErrorComponent,
+    LoginPage,
+} from "@pankod/refine-antd";
+import dataProvider from "@pankod/refine-simple-rest";
+import routerProvider from "@pankod/refine-react-router-v6";
+import "@pankod/refine-antd/dist/styles.min.css";
+
+import { PostList, PostCreate, PostEdit, PostShow } from "pages/posts";
+
+const API_URL = "https://api.fake-rest.refine.dev";
+
+// temporary set for refine-cluod-dev-tools
+window.localStorage.setItem(
+    "refine-cloud-token",
+    process.env.REACT_APP_REFINE_CLOUD_TOKEN as string,
+);
+
+const App: React.FC = () => {
+    return (
+        <Refine
+            LoginPage={LoginPage}
+            routerProvider={routerProvider}
+            dataProvider={dataProvider(API_URL)}
+            resources={[
+                {
+                    name: "posts",
+                    list: PostList,
+                    create: PostCreate,
+                    edit: PostEdit,
+                    show: PostShow,
+                    canDelete: true,
+                },
+            ]}
+            notificationProvider={notificationProvider}
+            Layout={Layout}
+            catchAll={<ErrorComponent />}
+            cloudConfig={{
+                baseUrl: "https://awesome.cloud.api.refine.dev",
```

Comment:
```suggestion
                baseUrl: "https://awesome.cloud.api.refine.dev",
```
Can we move it to `.env`?